app/controllers/goals_controller.rb

```diff
@@ -12,6 +12,22 @@ def create
     end
   end
 
+  def destroy
```

Comment:
After implementing that code,

```sql
SELECT  "goals".* FROM "goals" WHERE "goals"."id" = ? LIMIT 1  [["id", 109]]
```

Becomes

```sql
SELECT  "goals".* FROM "goals" WHERE "goals"."user_id" = ? AND "goals"."id" = ? LIMIT 1  [["user_id", 21], ["id", 110]]
```

Does this have any impact on response times?  In either case, you're only returning either one or zero items, but does a more specific query improve database efficiency?